Source/PeleLM_derive.cpp

```diff
@@ -135,7 +156,7 @@ void pelelm_dermolweight (const Box& bx, FArrayBox& derfab, int dcomp, int /*nco
           Yt[n] = in_dat(i,j,k,n+1) * rhoinv;
         } 
         EOS::Y2WBAR(Yt,Wbar);
-        der(i,j,k) = Wbar;
+        der(i,j,k+dcomp) = Wbar;
```

Comment:
it should be k,dcomp ?

Reply:
Yes, that was a stupid mistake.  Fixed below